examples/demo/lightning_representative_classification.py

```diff
@@ -10,8 +10,8 @@
 import torch
 
 # Dataset and machine learning model
-from examples.utils.datasets import BreastCancerDataset  # noqa: E402
-from examples.utils.ml_models import BreastCancerClassification  # noqa: E402
+from utils.datasets import BreastCancerDataset  # noqa: E402
```

Comment:
do we need the # noqa: E402?

Reply:
It tells the linter to be quiet about the import layout, which I am happy with, so I'd like to say yes.

Reply:
It seems strange that the linter complains specifically about this import line, but not the other ones. For this particular case, it should only be needed if there is code between the imports (https://www.flake8rules.com/rules/E402.html), but I can't see that here

Reply:
removed, seem to be a remnant of the past.